sunpy/map/sources/sdo.py

```diff
@@ -43,15 +42,23 @@ def processing_level(self):
 
     def _get_mpl_normalizer(self):
         """Returns a Normalize object to be used with AIA data"""
-        # byte-scaled images have most likely already been scaled
-        if self.data.dtype == np.uint8:
-            return None
-
-        mean = self.mean()
-        std = self.std()
-
-        vmin = max(0, mean - 3 * std)
-        vmax = min(self.max(), mean + 3 * std)
+        vmin = self.data.min()
+        vmax = self.data.max()
```

Comment:
These aren't necessary because `Normalize` (and derived classes) already defaults to going from the minimum to the maximum in the data array, which is what it is called on.  It is also only called on the whole array, so there's no weirdness with re-normalizations when cropping using the matplotlib GUI.